src/main/resources/application-klara.yml

```diff
@@ -0,0 +1,11 @@
+spring:
```

Comment:
Pls, don't push your config.yml file to the pull request. You can add it to the .gitignore file